code/datums/gamemodes/spy_theft.dm

```diff
@@ -334,8 +334,6 @@
 
 	station_bounties[/obj/item/disk/data/floppy/read_only/communications] = 2
 	station_bounties[/obj/item/disk/data/floppy/read_only/authentication] = 3
-	station_bounties[/obj/item/disk/data/floppy/manudrive/ai] = 2
```

Comment:
mb I'll revert this

Reply:
let me know if you need any help in imcoder